source/features/parse-backticks.tsx

```diff
@@ -4,46 +4,63 @@ import select from 'select-dom';
 import features from '../libs/features';
 import {parseBackticks} from '../libs/dom-formatters';
 
-function initGeneral(): void {
+function parseAll(): void {
 	for (const title of select.all([
 		'.commit-title', // `isCommit`
 		'.commit-desc', // `isCommit`, `isCommitList`, `isRepoTree`
-		'.commit-message', // Pushed commits in `isPRConversation`
-		'.message', // `isCommitList`, `isRepoTree`
+		'.commit-message', // Pushed commits in `isPRConversation`, `isCompare`, `isReleasesOrTags`
+		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
 		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
 		'[aria-label="Issues"][role="group"] .js-navigation-open', // `isDiscussionList`
 		'[id^=ref-issue-]', // Issue references in `isIssue`, `isPRConversation`
 		'[id^=ref-pullrequest-]', // PR references in `isIssue`, `isPRConversation`
 		'.TimelineItem-body > del, .TimelineItem-body > ins', // Title edits in `isIssue`, `isPRConversation`
-		'.js-pinned-issue-list-item > span' // Pinned Issues
-	])) {
+		'.js-pinned-issue-list-item > span', // Pinned Issues
+		'.pulse-section li', // `isPulse`
+		'.issues_labeled .text-gray-dark > a', // Newsfeed issues
+		'.commits blockquote', // Newsfeed commits
+		'.release-header', // `isReleasesOrTags` Headers
+		'.Box-row.js-navigation-item a.link-gray-dark', // `isGlobalDiscussionList`
+		'[data-channel^="check_suites"] a', // `isActions`
+		'.repository-content .pr-toolbar h2', // `isActions` run
+		'#wiki-wrapper h1' // `isWiki`
+	].map(selector => selector + ':not(.rgh-backticks-already-parsed)'))) {
+		title.classList.add('rgh-backticks-already-parsed');
 		parseBackticks(title);
 	}
 }
 
-// Highlight code in issue/PR titles on Dashboard page ("Recent activity" container)
+const observer = new MutationObserver(([{addedNodes}]) => {
+	parseAll();
+
+	// Observe the new ajaxed-in containers
+	for (const node of addedNodes) {
+		if (node instanceof HTMLDivElement) {
+			observer.observe(node, {childList: true});
+		}
+	}
+});
+
+// Highlight code in issue/PR titles on Dashboard page
 async function initDashboard(): Promise<void> {
 	await elementReady('.js-recent-activity-container', {stopOnDomReady: false});
 	for (const title of select.all('.js-recent-activity-container .text-bold')) {
 		parseBackticks(title);
 	}
+
+	observer.observe((await elementReady('#dashboard .news'))!, {childList: true});
 }
 
 features.add({
 	id: __featureName__,
 	description: 'Renders text in `backticks` in issue titles and commit titles/descriptions.',
 	screenshot: 'https://user-images.githubusercontent.com/170270/55060505-31179b00-50a4-11e9-99a9-c3691ba38d66.png',
 	include: [
-		features.isCommit,
-		features.isCommitList,
 		features.isDiscussionList,
-		features.isIssue,
-		features.isPRConversation,
-		features.isRepoTree,
-		features.isSingleFile
+		features.isRepo
 	],
 	load: features.onAjaxedPages,
-	init: initGeneral
+	init: parseAll
 });
 
 features.add({
```

Comment:
The listener already exists:

```js
init: () => onNewsfeedLoad(parseAll)
```

In a separate PR you might want to drop it from `hide-own-stars`

Reply:
Any other changes needws?

Reply:
It doesn't look like it at the moment

Reply:
The suggested code actually might not be enough since maybe it only covers the feed, not the "Recent Activity" box

Reply:
>The suggested code actually might not be enough since maybe it only covers the feed, not the "Recent Activity" box

But it would take away that MutationObserver
So make a new includes for it correct?

Reply:
And?

Reply:
Yes it does